package.json

```diff
@@ -32,47 +32,47 @@
   },
   "homepage": "https://github.com/tjoskar/ng-lazyload-image#readme",
   "peerDependencies": {
-    "@angular/core": ">=2.0.0",
-    "rxjs": ">=5.0.0"
+    "@angular/core": ">=6.0.0",
```

Comment:
Is this really necessary? `4.0.0` is still the current LTS version and I don't want to force everyone to upgrade.

Reply:
I agree some people may not want to upgrade. v6 is RC.3, so it should be official soon. The official release was supposed to be last week, but got delayed.

https://github.com/angular/angular/blob/master/docs/RELEASE_SCHEDULE.md

I think they only promised to maintain backwards compatibility with 1 previous version. So, once v6 is released, it should be compatible with v5, but nothing earlier.

I tried upgrading my app to v6 without changing ng-lazyload-image & had issues. I think the issues are actually related to rxjs & not angular.

If I upgrade my app to angular v6 but leave rxjs on 5.5.6 I get this error:
![image](https://user-images.githubusercontent.com/2002030/38517288-b2b23210-3bf6-11e8-971a-5b77def7e0e2.png)

I tried using 5.6.0-foward-compat.0 in my app & had issues with angular/cdk, not ng-lazyload-image. When I upgraded my app to 6.0.0-tactical-rc.1 the angular/cdk issues went away, but errors in ng-lazyload-image appeared.

It's possible I could leave my app on 6.0.0-tactical-rc.1 but put my fork of ng-lazyload-image on 5.6.0-foward-compat.0 & angular v4 or angular v5. I haven't tested that, but it's probably worth a shot.